CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * Publish `ERROR` events from `<RouteErrorBoundary>` so handler modules can react to them. Refs STCOR-455.
 * Dispatch `setOkapiReady` when all resources are loaded. Fixes STCOR-506.
 * Configure `swr`. Refs STCOR-516.
+* Callouts are not opened after page reload. Refs STCOR-518.
```

Comment:
This is a worrying change-log entry! Sure it should say that callouts now  _are_ opened after a page reload!